src/foundry/client/pixi/webgl/extensions/batch-renderer.d.mts

```diff
@@ -1,37 +1,78 @@
+import type { ToMethod } from "../../../../../types/helperTypes.d.mts";
+import type { InexactPartial } from "../../../../../types/utils.d.mts";
+
 export {};
 
+declare abstract class AnyBatchRenderer extends BatchRenderer {
+  constructor(arg0: never, ...args: never[]);
+}
+
 declare global {
+  namespace BatchRenderer {
+    type AnyConstructor = typeof AnyBatchRenderer;
+
+    type PackInterleavedGeometryFunction = ToMethod<
+      (
+        element: PIXI.IBatchableElement,
+        attributeBuffer: PIXI.ViewableBuffer,
+        indexBuffer: Uint16Array,
+        aIndex: number,
+        iIndex: number,
+      ) => void
+    >;
+
+    type PreRenderBatchFunction = (batchRenderer: typeof BatchRenderer) => void;
+
+    type BatchDefaultUniformsFunction = (maxTextures: number) => AbstractBaseShader.Uniforms;
```

Comment:
Since these are always properties on a class I'm okay giving the go ahead and adding `ToMethod` here. That way subclasses can do their regular (unsound) things haha.

Reply:
Done